nginx/ng/config.sls

```diff
@@ -4,6 +4,11 @@
 
 {% from 'nginx/ng/map.jinja' import nginx, sls_block with context %}
 
+{% if nginx.install_from_source %}
+/var/log/nginx:
```

Comment:
This sort of naming style is frowned upon, please use an identifier, then `- name: /var/log/nginx`.